index.js

```diff
@@ -12,21 +12,23 @@ const {
 const { getUsers } = require('./utils/storage');
 const { deleteUser } = require('./models/users');
 const {
-    ideasDonationSum,
-    ideasSumDifference,
-    createDonation,
-} = require('./businessRules/donations');
+    getTotalSumDonatedForIdea,
+    getIdeasSumDifference,
```

Comment:
Can you improve the `getIdeasSumDifference` name?

Reply:
Renamed. hopefully not to verbose. )